nix/flake/flakeref.go

```diff
@@ -67,6 +67,8 @@ type Ref struct {
 	// or "git". Note that the URL is not the same as the raw unparsed
 	// flake ref.
 	URL string `json:"url,omitempty"`
+
+	Raw string `json:"raw,omitempty"`
```

Comment:
could you add a comment for what Raw is, alongwith a few examples? 

That will be helpful, and also inline with the nice comments in the rest of the struct :)

Reply:
Alternatively, you could add a comment to point people to docblock of `parseRef` function